package.json

```diff
@@ -23,6 +23,7 @@
     "@types/react": "^18.2.8",
     "@types/react-dom": "^18.2.4",
     "@urql/next": "^1.0.0-beta.2",
+    "axios": "^1.7.7",
```

Comment:
Is there a way to do this using XMLHttpRequest so we can avoid the extra library?

Reply:
Changed to XMLHttpRequest and removed axios.